### ISSUE TITLE
Windows Defender usage is high when using si.battery() multiple times

### ISSUE THREAD

**Issue by davidacampos** [sebhildebrandt/systeminformation#732]
**Describe the bug**
In any version after 5.6.22, the usage of si.battery() is slower. Also, if placed in a loop with a 1 second wait between each call, it makes Windows Defender (Win 10) CPU usage spike (which might a consequence of the slowness). In general si.battery() calls seem 3x slower than pre 5.6.22.

**To Reproduce**
Hitting si.battery() multiple times... even if just once the slowness is obvious

**Current Output**
Every si.battery() call takes around 100ms if pre 5.6.22. It takes around 400ms if post 5.6.22

**Expected behavior**
Every si.battery() call to take the same and not spike Windows Defender cpu usage

**Environment (please complete the following information):**
 - systeminformation package version:
 - OS: Windows 10
 - Hardware: Intel NUC 10 with APC UPS

**Additional context**
I would assume there was some internal change on how that data is fetch after 5.7, which Windows Defender might be thinking is suspicious and/or is spawning a new shell every time that would also trigger it

**Comment by sebhildebrandt** [comment]
Hi @davidacampos ... yes you are right, back then, we added support for detecting multiple batteries and optimized design capacity and full capacity information for Windows. Since then, multiple `Get-WmiObject` calls are necessary to get all battery information. I am just wondering about the version you mentioned where we have this change ... As far as I see we did this change in version 5.7.5.
